pkg/client/accountsmgmt/v1/access_token_type.go

```diff
@@ -25,6 +25,11 @@ package v1 // github.com/openshift-online/uhc-sdk-go/pkg/client/accountsmgmt/v1
 type AccessToken struct {
 }
 
+// Empty returns true if the object is empty, i.e. no attribute has a value.
+func (o *AccessToken) Empty() bool {
+	return o == nil || (true)
```

Comment:
Why `|| true`?
Looks like degenerate case of `(x==nil && ... && true)` pattern, but means Empty() will always return true!

Reply:
This code is automatically generated, and the `true` at the end avoids having to handle different the last attribute of the object. But I forgot that there are objects that have no attributes, like this one. I will fix it.

Reply:
Does it make sense this struct is emtpy though?

Reply:
Hmm, for an empty struct always returning `Empty() == true` actually sounds correct 🤔

Reply:
It makes some sense, but not a lot. It is empty because in the API specification is defined as type-less _object_, and the tool I am using to generate this code doesn't support that yet.